build/azure-pipelines.yml

```diff
@@ -86,34 +86,16 @@ stages:
               arguments: '--configuration $(buildConfiguration) --no-restore -p:ContinuousIntegrationBuild=true'
           - script: |
               version="$(Build.BuildNumber)"
-              echo "varsion: $version"
+              echo "Version: $version"
 
               major="$(echo $version | cut -d '.' -f 1)"
-              echo "major version: $major"
+              echo "Major version: $major"
 
               echo "##vso[task.setvariable variable=majorVersion;isOutput=true]$major"
             displayName: Set major version
             name: determineMajorVersion
-          - task: PowerShell@2
-            displayName: Prepare nupkg
-            inputs:
-              targetType: inline
-              script: |
-                $umbracoVersion = "$(Build.BuildNumber)" -replace "\+",".g"
-                $templatePaths = Get-ChildItem 'templates/**/.template.config/template.json'
-
-                foreach ($templatePath in $templatePaths) {
-                  $a = Get-Content $templatePath -Raw | ConvertFrom-Json
-                  if ($a.symbols -and $a.symbols.UmbracoVersion) {
-                    $a.symbols.UmbracoVersion.defaultValue = $umbracoVersion
-                    $a | ConvertTo-Json -Depth 32 | Set-Content $templatePath
-                  }
-                }
-
-                foreach($csproj in Get-ChildItem -Recurse -Filter *.csproj)
-                {
-                  dotnet pack $csproj --configuration $(buildConfiguration) --no-build --output $(Build.ArtifactStagingDirectory)/nupkg
-                }
+          - script: dotnet pack $(solution) --configuration $(buildConfiguration) --no-build --property:PackageOutputPath=$(Build.ArtifactStagingDirectory)/nupkg
```

Comment:
As mentioned in https://github.com/dotnet/sdk/issues/30624#issuecomment-1432118204, the output option error introduced in .NET SDK 7.0.200 will be skipped entirely for the pack command in the next servicing release.

In the meantime, instead of looping over all csproj files and running multiple dotnet pack commands (even on projects that have `IsPackable=false`), we can pack the solution and set the specific output path property to avoid this error.